Municipalities (#11)
agrc/metadata-asset-tracking-tool: metadata/boundaries/municipal_boundaries.md

# Title

Utah Municipal Boundaries

## ID

6b60c759-74d6-4533-93f5-fed8cf32a748

## Brief Summary

> Statewide polygon dataset of municipal boundaries for cartography and approximate boundary identification.

Author: I'll mention that it's the authoritative source in the How Reliable Is The Dataset section.  After thinking about it, I was beginning to feel saying that the boundary dataset was for identifying boundaries felt a little silly and redundant. So, I changed it to "cartography and general analytic purposes".

### What does the dataset represent?

This dataset does not represent exact legal boundaries, but, rather a set of polygon boundaries used for general purposes that conforms to logical and administrative rules (e.g. no two cities or redevelopment areas may cover the same geographic extent).

### How was the dataset created?

This dataset was first created through a cooperative effort between the [Governor's Office of Planning and Budget](https://gopb.utah.gov/), [UDOT](https://udot.utah.gov/), the [Utah State Tax Commission](https://incometax.utah.gov/), and UGRC. Polygon features for each municipality were created using coordinate geometry [(COGO)](https://pro.arcgis.com/en/pro-app/latest/help/editing/introduction-to-cogo.htm) from legal descriptions. All features have been adjusted to the current Geographic Coordinate Database (GCDB) points.

> The original dataset incorporated municipalities in the state that were extant as of 1992. Updates for further annexations and disconnections have been completed with the cooperative efforts of the State Tax Commission, [Property Tax Division](https://propertytax.utah.gov/), and UGRC. Updates to the municipal boundaries proposed by counties or other entities must be certified through the [Lieutenant Governor's Office](https://ltgovernor.utah.gov/).

Reviewer: ```suggestion
The original dataset contained municipalities that were extant as of 1992. Since then, UGRC has updated it to reflect additional annexations and disconnections certified through the [Lieutenant Governor's Office](https://ltgovernor.utah.gov/) in cooperation with the State Tax Commission [Property Tax Division](https://propertytax.utah.gov/).
```

Author: Adjusted the wording a bit, let me know what you think!

Reviewer: So much better than my convoluted sentence. I love it.
